integration-tests/parse_files.sh

```diff
@@ -5,6 +5,7 @@ set -o nounset
 
 readonly CWD="$PWD"
 readonly TESTS_DIR="integration-tests/Dockerfiles"
+readonly BLACKLIST="./Dockerfiles/dockerfiles/nylas/sync-engine/Dockerfile"
```

Comment:
This file contains instructions that, while `docker build` can parse correctly, the docker file reference says it should not be possible. I'm sticking to the documentation